docs/commands/bundle.md

```diff
@@ -1,117 +1,163 @@
 ```
 
-### How to make a fully dereferenced bundle
+### Format
 
-<div class="warning">
-JSON output only works when there are no circular references.
-</div>
+#### Codeframe (default)
 
+```bash request
+openapi bundle pet.yaml store.yaml -o ./bundled --format=codeframe
+## equivalent to: openapi bundle pet.yaml store.yaml -o ./bundled
 ```
-openapi bundle --dereferenced --output dist --ext json openapi/openapi.yaml openapi/petstore.yaml
+
+```bash output
+...
 ```
 
-### Options
+Please note that the problems are displayed in the following format: `file:line:column`. For example, `petstore-with-errors.yaml:16:3`.
 
-#### Skip preprocessor
+Depending on the terminal emulator you use, it may be possible to directly click this indicator to edit the file in place.
 
-You may want to skip specific preprocessors upon running the command.
+#### Stylish
 
-```
-openapi bundle --skip-preprocessor=discriminator-mapping-to-one-of,another-example
+```bash request
+openapi bundle pet.yaml store.yaml -o ./bundled --format=stylish
 ```
 
-Learn more about [preprocessors](../resources/custom-rules.md).
+```bash output
+...
+```
 
-#### Skip rule
+In this format, `bundle` shows the file name, line number, and column where the problem occurred. However, the output is compressed and omits other contexts and suggestions.
 
-You may want to skip specific rules upon running the command.
+#### Json
 
+```bash request
+openapi bundle pet.yaml store.yaml -o ./bundled --format=json
 ```
-openapi bundle --skip-rule=no-sibling-refs,no-parent-tags
+
+```bash output
+bundling pet.yaml...
+{
+  "totals": {
+    "errors": 0,
+    "warnings": 0,
+    "ignored": 0
+  },
+  "version": "1.0.0-beta.54",
+  "problems": []
+}📦 Created a bundle for pet.yaml at bundled/pet.yaml 28ms.
+bundling store.yaml...
+{
+  "totals": {
+    "errors": 0,
+    "warnings": 0,
+    "ignored": 0
+  },
+  "version": "1.0.0-beta.54",
+  "problems": []
+}📦 Created a bundle for store.yaml at bundled/store.yaml 15ms.
 ```
 
-Learn more about [rules](../resources/custom-rules.md).
+In this format, `bundle` shows the result of bundling (including the number of errors and warnings and their descriptions) in JSON-like output.
+
+### Skip preprocessor, rule, or decorator
 
-#### Skip decorator
+You may want to skip specific preprocessors, rules, or decorators upon running the command.
 
-You may want to skip specific decorators upon running the command.
+```bash Skip preprocessors
+openapi bundle --skip-preprocessor=discriminator-mapping-to-one-of,another-example
+```
 
+```bash Skip rules
+openapi bundle --skip-rule=no-sibling-refs,no-parent-tags
 ```
+
+```bash Skip decorators
 openapi bundle --skip-decorator=generate-code-samples,remove-internal-operations
 ```
 
-Learn more about [decorators](../resources/custom-rules.md).
+:::success Tip
+To learn more about preprocessors, rules, and decorators, please refer to the [custom rules page](../resources/custom-rules.md).
```

Comment:
question: This same sentence is near the start of the page. Do we need it both places?

Reply:
I would remove it from here, but l'm interested in what @bandantonio thinks about this

Reply:
@adamaltman The link appears here because if you want to know more information about custom rules, you have to scroll to the top and search for that link somewhere in the text. When you can immediately proceed to the extended section right from the place where some basics are described, it is actually a good idea. Anyway, if you or @skadinna think that it is redundant, I will remove one of them.

Reply:
I think we can leave both tips as is - they look pretty organic.

I believe we can merge this PR.